src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -3426,6 +3426,442 @@ exports[`StoryPromo Container should render multiple Index Alsos correctly for c
+}
+
+@supports (grid-template-columns:fit-content(200px)) {
+  .c0 {
+    display: grid;
+    grid-template-columns: repeat(6,1fr);
+    grid-column-gap: 0.5rem;
+  }
+}
+
+@media (min-width:80rem) {
+  .c1 {
+    width: 33.33%;
+  }
+}
+
+@supports (grid-template-columns:fit-content(200px)) {
+  .c1 {
+    display: block;
+    width: initial;
+    grid-column: 1 / span 2;
+  }
+}
+
+@media (min-width:25rem) {
+  .c4 {
+    position: absolute;
+    bottom: 0;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c6 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c6 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c6 {
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c8 {
+    font-size: 0.9375rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c8 {
+    font-size: 0.875rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (max-width:37.4375rem) {
+  .c8 {
+    display: none;
+    visibility: hidden;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c5 {
+    padding: 0 1rem;
+  }
+}
+
+@media (min-width:80rem) {
+  .c5 {
+    width: 66.67%;
+  }
+}
+
+@supports (grid-template-columns:fit-content(200px)) {
+  .c5 {
+    display: block;
+    width: initial;
+    padding: initial;
+    grid-column: 3 / span 4;
+  }
+}
+
+<div
+  class="c0"
+>
+  <div
```

Comment:
what are all these divs for??

Reply:
`<div class="c0">` - For outer Grid of Story Promo
`<div class="c1">` - Part of the Grid that takes up 1/3 of the outer grid and displays an Image
`<div class="c2">` - Image Contents Wrapper
`<div class="c3">` - Image Placeholder (the one that contains an actual image inside)
`<div class="c4">` - Inline Media Indicator (this gets rendered in DOM even when the Story Promo doesn't display a media indicator, that's just the way it works in Simorgh now)
`<div class="c5">` - Part of the Grid that takes up the remainder 2/3 of the outer grid and displays a Headline+Summary